testnet/tools/nns-tools/add-release-to-changelog.sh

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+
+NNS_TOOLS_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
+source "$NNS_TOOLS_DIR/lib/include.sh"
+
+help() {
+    print_green "
+Usage: $0 <PROPOSAL_ID>
+    PROPOSAL_ID: The ID of a recently executed Governance backend canister upgrade proposal.
+
+    Moves the pending new changelog entry from unreleased_changelog.md to CHANGELOG.md.
+
+    The commit that the proposal is baded on MUST be currently check out.
+    Otherwise, this script will do nothing, and terminate with a nonzero exit
+    status.
+
+" >&2
+    exit 1
+}
+
+PROPOSAL_ID="${1}"
+
+cd $(repo_root)
+PWD=$(pwd)
+
+# Fetch the proposal.
+print_purple "Fetching proposal ${PROPOSAL_ID}..." >&2
+PROPOSAL_INFO=$(
+    __dfx --quiet \
+        canister call \
+        --ic \
+        --candid rs/nns/governance/canister/governance.did \
+        $(nns_canister_id governance) \
+        get_proposal_info "(${PROPOSAL_ID})" \
+        | idl2json
+)
+# Unwrap.
+LEN=$(echo "${PROPOSAL_INFO}" | jq '. | length')
+if [[ "${LEN}" -ne 1 ]]; then
+    print_red "Unexpected result from the get_proposal_info method:" >&2
+    print_red "Should have one element, but has ${LEN}" >&2
+    exit 1
+fi
+PROPOSAL_INFO=$(echo "${PROPOSAL_INFO}" | jq '.[0]')
+
+# Assert was executed.
+EXECUTED_TIMESTAMP_SECONDS=$(echo "${PROPOSAL_INFO}" | jq '.executed_timestamp_seconds | tonumber')
+if [[ "${EXECUTED_TIMESTAMP_SECONDS}" -eq 0 ]]; then
+    print_red "Proposal ${PROPOSAL_ID} exists, but was not successfully executed." >&2
+    exit 1
+fi
+SECONDS_AGO=$(($(date +%s) - "${EXECUTED_TIMESTAMP_SECONDS}"))
+EXECUTED_ON=$(
+    date --utc \
+        --date=@"${EXECUTED_TIMESTAMP_SECONDS}" \
+        --iso-8601
+)
+print_purple "Proposal ${PROPOSAL_ID} was executed ${SECONDS_AGO} seconds ago." >&2
+
+# Extract which canister was upgraded, and to what commit.
+TITLE=$(echo $PROPOSAL_INFO | jq -r '.proposal[0].summary' | head -n 1)
```

Comment:
I think $PROPOSAL_INFO needs to be quoted, e.g.
```suggestion
TITLE=$(echo "${PROPOSAL_INFO}" | jq -r '.proposal[0].summary' | head -n 1)
```